internal/syncd/integrations/ecr/git.go

```diff
@@ -0,0 +1,57 @@
+package ecr
+
+import (
+	"context"
+
+	"github.com/google/go-github/v26/github"
+	"golang.org/x/oauth2"
+)
+
+type GitHub struct {
+	client       *github.Client
+	Organization string
+	Repository   string
+}
+
+func NewGitHub(ctx context.Context, token string, org string, repo string) GitHub {
+	tokenSource := oauth2.StaticTokenSource(
+		&oauth2.Token{AccessToken: token},
+	)
+
+	client := GitHub{
+		client:       github.NewClient(oauth2.NewClient(ctx, tokenSource)),
+		Organization: org,
+		Repository:   repo,
+	}
+
+	return client
+}
+
+func (c GitHub) UpdateFile(msg string, branch string, path string, fileContent []byte) (*github.RepositoryContentResponse, error) {
```

Comment:
Should test cases be written for these functions? They do not have much logic to test, however, the code coverage would be greatly improved on this PR by adding them.